Fashionz/index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+    <title>Fashionz</title>
+    <meta name="viewport" content="width=device-width", initial-scale="1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="assets/styles.css">
+</html>
+
+<body class="body-bg-color">
+    <!----------START :: Header section--------->
+    <header>
+        <div class="fashionz-logo">
+            <div><img src="assets/images/logo.png" alt=""></div>
+        </div>
+        <div class="fashionz-menu-icon">
+            <div class="fashionz-menu-dropdown">
+                <div><a href="">My Account</a></div>
+                <div><a href="">My Orders</a></div>
+                <div><a href="">My Wish List</a></div>
+                <div><a href="">Logout</a></div>
+            </div>
+        </div>
+    </header>
+    <!-----------END :: Header section----------->
+
+     <!----------START :: Main section--------->
+    <main>
+        <div class="main-section-container">
+            <!-- Dress thumbnail image -->
+            <div class="dress-thumbnail">
+                <div>
```

Comment:
Why do we need so many div across ? , We need to make the code more DRY across the app

Reply:
Review comments resolved